astropy/tests/pytest_plugins.py

```diff
@@ -147,8 +147,9 @@ def pytest_configure(config):
                           allow_astropy_data=config.getoption('remote_data') == 'astropy')
 
     doctest_plugin = config.pluginmanager.getplugin('doctest')
-    if (doctest_plugin is None or config.option.doctestmodules or not
-            (config.getini('doctest_plus') or config.option.doctest_plus)):
+    if (doctest_plugin is None or config.option.doctestmodules
+            or not (config.getini('doctest_plus').lower() in ['true', 'enabled']
```

Comment:
(needless to say, I'm happy to change it back to the old way if you prefer. I like it more this way, but also like my editor lacking the big red lines from flake8. Apparently I can't have both :) ).